README.md

```diff
@@ -39,5 +39,19 @@ Encrypted vars can be added to `.travis.yml` using the travis cli tools as decri
 Currently configured with GITHUB_TOKEN value.
 
 
-### To Production (TBC)
+### To Production
+The production is hosted using [now.sh](https://now.sh).
+The process is manual and requires deploying two now apps. You can do that by installing now and deploying the separate now.json files.
+To speed this up you can make sure now is installed globally before running npx `yarn global add now`.
```

Comment:
```suggestion
To speed this up you can make sure `now` is installed globally before running npx `yarn global add now`.
```
I'd just keep `now` in code tags to avoid confusion. Also, this whole phrase should be after the previous and add a new line after.